package.json

```diff
@@ -9,17 +9,19 @@
     "travis": "istanbul --include-all-sources cover _mocha -- --reporter spec"
   },
   "dependencies": {
-    "httperrors": "1.1.0"
+    "httperrors": "2.1.0"
   },
   "devDependencies": {
-    "coveralls": "^2.11.6",
-    "express": "4.13.3",
-    "istanbul": "0.4.1",
+    "coveralls": "2.11.15",
+    "express": "4.14.0",
+    "istanbul": "0.4.5",
     "magicpen-media": "1.5.1",
-    "messy": "6.6.2",
-    "mocha": "2.3.4",
-    "unexpected": "10.3.1",
-    "unexpected-express": "8.0.0",
-    "unexpected-messy": "5.0.0"
+    "messy": "6.12.2",
+    "mocha": "3.2.0",
+    "sinon": "1.17.7",
+    "unexpected": "10.21.1",
+    "unexpected-express": "8.3.3",
+    "unexpected-messy": "6.2.0",
+    "unexpected-sinon": "10.5.1"
```

Comment:
There were peer dependencies conflicting with unexpected. If you want to trace the exact difference I can move the general update to master.